README.md

```diff
@@ -228,7 +206,6 @@ The `Close` redeemer allows the owner (signified by the address' staking credent
     - pubkey must sign
     - script must be executed in the same tx
 
-:important: In the previous version, the `Close` redeemer also checked the transaction outputs to make sure any new outputs to the script were valid. This feature was not necessary and was removed. The `Close` redeemer's purpose is to close an address, not add new positions.
```

Comment:
You deleted this. This is an importance change.

Reply:
even if it is an important change, should keep versions changes out of the README. Thats what the changelog is for. I can add this to the changelog?

Reply:
It already is in the changelog. IMO breaking changes should be mentioned in the documentation too since this is meant to be accessible to everyone. For example, the fact the script no longer converts ADA -> Lovelace is also mentioned.

Reply:
i disagree but I'll mention it in a note for now. can delete later